### PR TITLE
 Init general layout

### PR DESCRIPTION
**Pull request by huynt-3640**
## Related Tickets
- [82736](https://edu-redmine.sun-asterisk.vn/issues/82736)

## WHAT (optional)
- Layout cho home page

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
image
image

### CODE REVIEW

**Comment by huynt-3640** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'switch_language/:locale', to: 'application#switch_language', as: :switch_language
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'home/index'
+  root 'home#index'
```
Comment (ghost) [inline config/routes.rb]: Dùng " 
Bọc các routes trong scope của i18n

Comment (huynt-3640) [reply config/routes.rb]: Vâng ạ

**Comment by huynt-3640** [comment]
ready

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  padding: 1rem 1rem 2rem 1rem;
+  margin: 1rem 0.5rem;
+  border-radius: 8px;
+  border: 1px solid #ecf0f1;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  
+}
+.btn-apply {
+  background-color: #27ae60;
+  color: #fff;
```
Comment (ghost) [inline app/assets/stylesheets/home.scss]: Đặt biến cho màu để tái sử dụng nhé

**Review by ghost — COMMENTED** [review]
cmt

**Comment by huynt-3640** [comment]
ready

**Pull request merged**